src/config/configuration.f90

```diff
@@ -305,20 +303,12 @@ subroutine copy_array_i(input, output, invalid)
         output(:n)  = input(:n)
     end subroutine copy_array_i
 
-    subroutine copy_array_weights(input, output, n, default)
+    subroutine copy_array_weights(input, output, n)
```

Comment:
If this is ready to merge, can you add a small doxygen style header comment for this subroutine: 
```
!>------------------------------------------------
!! Description
!!
!!------------------------------------------------
```